lib/livebook/teams/connection.ex

```diff
@@ -44,16 +44,20 @@ defmodule Livebook.Teams.Connection do
       {:ok, conn, websocket, ref} ->
         send(data.listener, :connected)
         send(self(), {:loop_ping, ref})
+        Logger.info("Teams WebSocket connection - established")
```

Comment:
@josevalim I see you suggested this one to change to `info` but the rest you left as `warning`. Any reason? 

Is that because this one is about a successful operation, and the others are error scenarios?

Reply:
Correct.